src/command.c

```diff
@@ -199,15 +201,37 @@ handle_command_stop (const struct subcommand *sub,
 
 	g_free_node(root);
 
+	/* Check for devices which were passed through to the clear container
+	 * If found, bind it back to the original host driver and change the
+	 * interface back to the child process' namespace
+	 */
+        if (state->devices) {
+                for (index=0; index<g_slist_length(state->devices); index++) {
+			struct cc_oci_device* device = (struct cc_oci_device *)
+				g_slist_nth_data(state->devices, index);
+
+                        if (!cc_oci_bind_host(device))
+				g_debug("failed to bind device %s back to host", device->bdf);
+
+			/* brute-force hacks for changing the interface to
+			 * child process network namespace
+			 */
+			if (!cc_oci_switch_iface_to_container(device, state->pid))
+				g_debug ("failed to switch device %s to container",device->bdf);
```

Comment:
Similar logic to above -- I think its best to finish looping and handling the other devices.